src/components/MovieView/index.tsx

```diff
@@ -7,11 +7,24 @@ import { ETypeView } from "@constants/typeView";
 import { useAppSelector } from "@store/hooks";
 import { IMovieViewProps } from "./types";
 import { selectFilterCurrentTypeView } from "@store/filter/selectors";
+import { PaginationWrapper } from "./styles";
 
-const MovieView = ({ movies }: IMovieViewProps) => {
+const MovieView = ({
+  movies,
+  currentPage,
+  totalPages,
+  onPageChange,
+}: IMovieViewProps) => {
   const { t } = useTranslation();
   const movieTypeView = useAppSelector(selectFilterCurrentTypeView);
 
+  const handlePageChange = (
+    event: React.ChangeEvent<unknown>,
+    page: number
+  ) => {
```

Comment:
если не используешь параметр, можно заменить его на `_`
```suggestion
  const handlePageChange = (
    _,
    page: number
  ) => {
```

Reply:
`Параметр "_" неявно имеет тип "any".ts(7006)`
Поэтому я название переменной поменяю, а тип оставлю

Reply:
да, не вопрос